src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -610,7 +610,11 @@ public function load(array $configs, ContainerBuilder $container): void
         $container->registerForAutoconfiguration(AssetCompilerInterface::class)
             ->addTag('asset_mapper.compiler');
         $container->registerAttributeForAutoconfiguration(AsCommand::class, static function (ChildDefinition $definition, AsCommand $attribute, \ReflectionClass $reflector): void {
-            $definition->addTag('console.command', ['command' => $attribute->name, 'description' => $attribute->description ?? $reflector->getName()]);
+            $definition->addTag('console.command', [
+                'command' => $attribute->name,
+                'description' => $attribute->description ?? $reflector->getName(),
+                'help' => $attribute->help,
```

Comment:
What's the benefit of calling setHelp in the service definition of the command? Can't we only rely on the attribute? Eg the default `Command::getHelp()` would read the attribute and done?
Having the help as a command tag attribute allows overriding the class attribute. But is there a use case for that?

Reply:
This is only necessary for invokable commands that do not extend the `Command` class.